WalletWasabi/Services/WalletService.cs

```diff
@@ -1351,9 +1343,7 @@ public void RefreshCoinHistories()
 						var result = string.Join(
 							", ",
 							GetClusters(coin, new List<SmartCoin>(), lookupScriptPubKey, lookupSpenderTransactionId, lookupTransactionId)
-							.SelectMany(x => x.Label
-								.Split(',', StringSplitOptions.RemoveEmptyEntries)
-								.Select(y => y.Trim()))
+							.SelectMany(x => x.Label.Labels)
 							.Distinct());
```

Comment:
In the `SmartLabel` class there is a `.Distinct(StringComparer.OrdinalIgnoreCase)`. Shouldn't we use the same here? I mean, if I have `Adam` and `ADAM` should we display both or only one?